openedx/stanford/lms/urls.py

```diff
@@ -19,8 +20,7 @@
 if settings.FEATURES.get('ENABLE_SUPERUSER_LOGIN_AS'):
     urlpatterns += [
         url(
-            r'^su_login_as/(?P<username>[\w.@+-]+)/?$',
-            'student.views.superuser_login_as',
-            name='impersonate',
+            r'',
+            include('openedx.stanford.djangoapps.superuser_login_as.urls'),
```

Comment:
shouldn't the url include be the first thing in the `url()` function?

Reply:
It's an `include`; it doesn't have a url itself.

Reference `lms/urls.py`

Reply:
oh ok, didn't know it worked like that.